exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -178,34 +177,14 @@ func getPromMetricName(metric *otlp.Metric, ns string) string {
 		return ""
 	}
 
-	// if the metric is counter, _total suffix should be applied
-	_, isCounter1 := metric.Data.(*otlp.Metric_DoubleSum)
-	_, isCounter2 := metric.Data.(*otlp.Metric_IntSum)
-	isCounter := isCounter1 || isCounter2
-
-	b := strings.Builder{}
-
-	b.WriteString(ns)
-
-	if b.Len() > 0 {
-		b.WriteString(delimiter)
+	var name string
+	if len(ns) > 0 {
+		name = ns + delimiter + metric.GetName()
```

Comment:
Consider inlining the `_`, I think it would improve readability

Reply:
Yeah, this was sticking with the previous style, but it was only used here and one other place so I've just removed it entirely.  I've also simplified this a bit further by initializing `name := metric.GetName()` since that was used in either branch and allowed removing the `else` branch.